_data/projects/opencensus-go.yml

```diff
@@ -0,0 +1,22 @@
+name: opencensus-go
+desc: A stats collection and distributed tracing framework
```

Comment:
Can these descriptions be a bit more distinct to help them stand out? Aside from the `-{language}` suffix and a couple of tags each of these projects is almost identical.